news/2023/2023-12-16-nat-gazette-dec-2023.md

```diff
@@ -0,0 +1,107 @@
+
+Fast-forward five years later, the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) has decided to reinstate the Gazette with similar goals in mind — to connect with the community and issue updates on a periodic basis!
+
+Without further ado, let's get to the nitty-gritty of this month's series of updates, shall we?
+
+## Updates
+
+In order to better organize ourselves and give users some time to adapt, the NAT has opted to deploy updates in batches. Once enough changes have been discussed and approved, these will be merged in one fell swoop. The Gazette will try to keep up with this sporadic update schedule.
+
+### Within the Ranked section
+
+#### Blocking Qualified maps with pending posts
+
+Qualified beatmaps with open suggestion or problem stamps in the discussion board will **not** be able to be Ranked, and the timer will not stop until the posts are either closed or the map gets disqualified to address the points made.
+
+In the case of a beatmap reaching its supposed ranking time and there are unresolved suggestions or problems, a NAT member will shortly step in and either resolve the post or disqualify the map depending on the validity of the concerns raised.
+
+#### The RC cleanup project
+
+There's currently a community-driven [Ranking Criteria cleanup project](https://osu.ppy.sh/community/forums/topics/1836226) in the works which aims to overhaul the current RC to make it much clearer and more accessible in general. If you believe you can contribute something to it, we encourage to give it a shot to help the team out!
+
+#### Ranking Criteria changes
+
+- **General:** Changes on how the `Featured Artist` tag should be used, and a special mention on how to credit AI-generated media in the beatmap's description.
+- **osu!:** Clarification on how to use hitsounds on actively clicked parts of objects.
+- **osu!taiko:** Relaxed the rest moment guideline for Kantan and Futsuu difficulties from 16/1—20/1 to 32/1—36/1.
+- **osu!catch:** Changed the CS and HP guidelines for Rain and Overdose difficulties respectively, and an extra guideline on how to use higher-snapped hyperdashes was added.
+- **osu!mania:** Key modes over 10 keys are now rankable so long they follow an allowed [playstyle](/wiki/Beatmapping/osu!mania_10K_plus_playstyles).
+
+### Within the Beatmap Nominators
+
+#### Changes to activity requirements
+
+Beatmap Nominators rejoining in good terms are now expected to nominate their mode's [minimum activity requirements](/wiki/People/Beatmap_Nominators/Rules#requirements) in the first 30 days since their rejoin. Make sure to take this into consideration before asking to join back into the team!
```

Comment:
```suggestion
Beatmap Nominators rejoining in good terms are now expected to reach their mode's [minimum activity requirements](/wiki/People/Beatmap_Nominators/Rules#requirements) in the first 30 days since their rejoin. Make sure to take this into consideration before asking to join back into the team!
```